client/services/bikes.tsx

```diff
@@ -46,6 +45,31 @@ const addBike = async (params) => {
     }
 };
 
+const makeBikeUnavailable = async (bikeId) => {
+    try {
+        const response = await axios.patch(`/bike/${bikeId}`, {
+            status: 'unavailable',
+            tags: []
```

Comment:
this is also a good solution for tags. I tried updating tags in patchBikes on the backend to:

tags: ('tags' in request.body) ? [...new Set([...currentBike.tags, ...request.body.tags])] : currentBike.tags
tags: (request.body.tags) ? [...new Set([...currentBike.tags, ...request.body.tags])] : currentBike.tags

but neither worked

Reply:
That was my lazy solution lol...it definitely works though! Maybe it's good enough for now?